chalicelib/checks/wrangler_checks.py

```diff
@@ -1063,15 +1062,15 @@ def core_project_status(connection, **kwargs):
         check.status = 'PASS'
         check.summary = 'All CGAP Core items are shared.'
         check.description = ('All CGAP Core items are shared:'
-                             ' {}'.format(item_type)) 
```

Comment:
Seems to me like this line break might not be needed at all, though I didn't check the line length carefully. But if it is needed, I suggest:
```
        check.description = ('All CGAP Core items are shared: {}'
                             .format(item_type))
```
There is no requirement to have the string on the same line as the method call, as long as you're inside a parenthetical context (or connected by a backslash+newline break, though I abhor those) to clarify that you're still parsing the previous expression.